mecsimcalc/spreadsheet_utils.py

```diff
@@ -108,42 +110,39 @@ def print_dataframe(
 
     Creates an HTML table from a pandas DataFrame and optionally provides a download link for the table.
 
-    # Parameters
-
+    Parameters
+    ----------
     df : pd.DataFrame
-        The DataFrame to be converted.
+        The DataFrame to be converted into an HTML table.
     download : bool, optional
-        If True, the function also provides a download link. (Defaults to False)
+        If True, the function also provides a download link for the table. Defaults to False.
     download_text : str, optional
-        The text to be displayed on the download link. (Defaults to "Download Table")
+        The text to be displayed on the download link. Defaults to "Download Table".
     download_file_name : str, optional
-        The name of the downloaded file. (Defaults to "myfile")
+        The name of the file to be downloaded. Defaults to "myfile".
     download_file_type : str, optional
-        The file type of the download file. xlsx or csv (Defaults to "csv")
-
-    # Returns
-
-    * `Union[str, Tuple[str, str]]` :
-        * `download | False` : str - Returns the HTML table as a string.
-        * `download | True` : Tuple[str, str] - Returns a tuple containing the HTML table and the HTML download link as strings.
-
-    # Examples
-    ** Without Download Link **
-    >>> input_file = inputs["input_file"]
-    >>> df = msc.input_to_dataframe(input_file)
-    >>> table = msc.print_dataframe(df)
-    >>> return {
-        "table": table
-    }
-
-    ** With Download Link for excel file **
-    >>> input_file = inputs["input_file"]
-    >>> df = msc.input_to_dataframe(input_file)
-    >>> table, download_link = msc.print_dataframe(df, download=True, download_file_type="xlsx")
-    >>> return {
-        "table": table,
-        "download_link": download_link
-    }
+        The file type of the download file. Can be "xlsx" or "csv". Defaults to "csv".
+
+    Returns
+    -------
+    Union[str, Tuple[str, str]]
+        If `download` is False, returns the HTML table as a string.
+        If `download` is True, returns a tuple containing the HTML table as a string and the HTML download link as a string.
+
+    Examples
+    --------
+    Without Download Link:
```

Comment:
The reason why the original was in this format:
```

>>> input_file = inputs["input_file"]
>>> df = msc.input_to_dataframe(input_file)
```

is because this is how users would normally use the function. 

Changing it to: 
`df = pd.DataFrame({'A': [1, 4], 'B': [2, 5], 'C': [3, 6]})`
makes it harder for new users to understand. Especially non experienced programmers

Reply:
didn't pay attention to that, sorry. I go through them again and reset examples to original format.